src/coreclr/vm/amd64/AsmHelpers.asm

```diff
@@ -721,6 +728,7 @@ NESTED_ENTRY _more_stack, _TEXT
         ;
         ; Save integer parameter registers.
         ;
+        save_reg_postrsp    r11, 68h
```

Comment:
This needs to be copied to ResumeSuspendedThreadHelper2 to keep them identical

Reply:
Eh, true, although it doesn't actually matter for correctness.